readthedocs/settings/base.py

```diff
@@ -621,7 +621,7 @@ def USE_PROMOS(self):  # noqa
             '': {  # root logger
                 'handlers': ['debug', 'console'],
                 # Always send from the root, handlers can filter levels
-                'level': 'DEBUG',
+                'level': 'INFO',
```

Comment:
If the root logger filters by INFO, can we set DEBUG level in the handler later? With this change, is DEBUG logging under `debug.log` file still working?

Reply:
We want to be able to log DEBUG messages, but only from our application. So we want DEBUG to be output, but only sent via the RTD app, not all libraries.